__tests__/internal/move-after-handle-request/route-handlers/function-handler/basic-test.js

```diff
@@ -7,13 +14,19 @@ describe("Integration | Route handlers | Function handler", () => {
     server = new Server({
       environment: "development",
       models: {
-        user: Model.extend({})
+        user: Model.extend({}),
+        post: Model.extend({
+          author: belongsTo()
+        }),
+        author: Model
```

Comment:
Could you get rid of all the changes to this file since we're not using it anymore?